config/generateRunFile.coffee

```diff
@@ -311,6 +311,14 @@ generateDev = (KONFIG, options) ->
       set +o errexit
     }
 
+    function runMongoDocker() {
+        docker run -d -p 27017:27017 --name=mongo koding/mongo:2016-10-11
```

Comment:
I think it'd be good to use latest tag for development environment. What do you think?